.editorconfig

```diff
@@ -26,6 +26,14 @@ indent_size = 2
 insert_final_newline = false
 charset = utf-8-bom
 
+[project.json]
+insert_final_newline = false
+charset = utf-8-bom
+
+[*.nuget.targets]
+insert_final_newline = false
+charset = utf-8-bom
+

```

Comment:
This is what NuGet does to these files, so better to embrace it, as we do with `*.{csproj,vbproj}`.

Note that I couldn't define these settings in one section, as the EditorConfig docs state you can do, e.g.

```yaml
[*.{csproj,vbproj},project.json,*.nuget.targets]
```

For some reason it doesn't work. I guess this is a bug in the VS EditorConfig extension.